src/sentry/grouping/fingerprinting/__init__.py

```diff
@@ -466,3 +493,38 @@ def visit_key(self, node, children):
     def visit_quoted_key(self, node, children):
         # leading ! are used to indicate negation. make sure they don't appear.
         return node.match.groups()[0].lstrip("!")
+
+
+def _load_configs():
+    if not CONFIGS_DIR.exists():
+        logger.warning(
+            "Failed to load Fingerprinting Configs, invalid _config_dir: %s",
+            CONFIGS_DIR,
+        )
+
+    configs = {}
+
+    for config_file_path in sorted(CONFIGS_DIR.glob("**/*.txt")):
+        config_name = config_file_path.parent.name
+        configs.setdefault(config_name, [])
+
+        try:
+            with open(config_file_path) as config_file:
+                str_conf = config_file.read().rstrip()
+                configs[config_name].extend(FingerprintingRules.from_config_string(str_conf).rules)
+        except InvalidFingerprintingConfig as ex:
+            logger.warning(
+                "Fingerprinting Config %s Invalid: %s",
+                config_file_path,
+                ex,
+            )
+        except Exception as ex:
+            logger.warning(
+                "Failed to load Fingerprinting Config %s: %s",
+                config_file_path,
+                ex,
+            )
+    return configs
+
+
+FINGERPRINTING_BASES = _load_configs()
```

Comment:
I'm not sure if we should do this on app load/import. Should we have a `get_fingerprinting_bases` function that loads and caches them on demand instead?

Reply:
That was my initial approach, but then I made it be as similar to Enhancements as possible, and there the only thing _cached_ is loding custom ones from DB. 

This is basically slighly more flexible version of https://github.com/getsentry/sentry/blob/a7a33951539f03f306380b7a454204e0ef24c1a3/src/sentry/grouping/enhancer/__init__.py#L632-L646